src/screens/Services/Services.js

```diff
@@ -253,22 +252,12 @@ class ServicesScreen extends React.Component<Props> {
           const email = user?.email;
           const address = this.getCryptoPurchaseAddress();
           if (address === null) return;
-
-          Alert.alert(
-            t('servicesContent.ramp.assetDecisionAlert.title'),
-            t('servicesContent.ramp.assetDecisionAlert.description'),
-            [
-              {
-                text: t('servicesContent.ramp.assetDecisionAlert.actionNonPlr'),
-                onPress: () => this.tryOpenCryptoPurchaseUrl(rampWidgetUrl(address, email)),
-              },
-              {
-                text: t('servicesContent.ramp.assetDecisionAlert.actionPlr'),
-                onPress: () => this.tryOpenCryptoPurchaseUrl(rampWidgetUrl(address, email, true)),
-              },
-            ],
-            { cancelable: true },
-          );
+          navigation.navigate(ADD_CASH, {
+            onSubmit: (values: AddCashParam) => {
```

Comment:
Important
Passing onSubmit callback requires all callers to create this kind of handler which hinders screen re-usage across the app. Pls move body of `this.tryOpenCryptoPurchaseUrl(rampWidgetUrl(address, email, fiatCurrency, fiatValue));` to AddCash screen

Reply:
Intent here is reusability for tryOpenCryptoPurchaseUrl method, any change demands changes in 2 files. and if any user misses then it will lead to a bug.

Reply:
@IAmKio @mdjastrzebski For SENDWYRE_INPUT also onSubmit callback is used(Services.js line 274). Do we need to change this also?

Reply:
Done